front/app/containers/HomePage/VerificationOnboardingStep.tsx

```diff
@@ -48,9 +45,7 @@ const VerificationOnboardingStep = ({
   contentTimeout,
   contentDelay,
 }: Props) => {
-  const theme: any = useTheme();
-  const handleSkip = () => onSkip(onboardingCampaigns.name);
-  const handleAccept = () => onAccept(onboardingCampaigns.name);
```

Comment:
I deel with passing the campaign name now in the parent (SignedInHeader.tsx) instead of here, which solves the issue.